source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -0,0 +1,189 @@
+===========================
+Update Arrays in a Document
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to update arrays in a document.
+
+To update an array, you must do the following:
+
+- Specify the update you want to perform
+- Specify what array elements to apply your update to
+- Perform an update operation using these specifications
+
+Sample Document
+~~~~~~~~~~~~~~~
+
+The following sections feature examples that update this sample
+document: 
+
+.. code-block:: json
+
+    { "_id": 1, "color": "green", "qty": [8, 12, 18] }
+
+The examples on this page use the ``findOneAndUpdate()`` method of the
+``MongoCollection`` class to retrieve and update our document. Each
+example uses an instance of the ``FindOneAndUpdateOptions`` class to
+configure the method to return the document after the update occurs. For
```

Comment:
```suggestion
have MongoDB retrieve the document after the update occurs. For
```
Suggestion: I think saying "configure the method" implies that the method has access to both the document in the state before the update and the state after the update. I think saying that "have mongodb retrieve" gives a clearer picture of what information the method has at what time.